src/overview/epics.js

```diff
@@ -1,6 +1,6 @@
 import 'rxjs/add/operator/debounceTime'
 import 'rxjs/add/operator/map'
-
+import { combineEpics } from 'redux-observable'
```

Comment:
Sorry it was a remainder of the hack I put in place to go about the ```...Objects.values(overview.epics)``` error. I am really sorry I left it there I will fix it right now.